pinot-core/src/main/java/org/apache/pinot/core/query/reduce/BaseReduceService.java

```diff
@@ -51,12 +51,15 @@ public abstract class BaseReduceService {
   protected final ExecutorService _reduceExecutorService;
   protected final int _maxReduceThreadsPerQuery;
   protected final int _groupByTrimThreshold;
+  protected final int _minGroupByTrimSize;
```

Comment:
To be consistent with server config name, suggest naming it `_minGroupTrimSize`, same for other places